examples/ex111_chebyshev.jl

```diff
@@ -14,6 +14,7 @@ diffusion = GalleryOperator("diffusion", p + 1, p + 1, mesh)
 
 # Chebyshev smoother
 chebyshev = Chebyshev(diffusion)
+setchebyshevtype(chebyshev, ChebyshevType.fourth)
```

Comment:
I'd prefer to use 
```julia
chebyshev.type = ChebyshevType.forth
```
here. You can modify `function Base.setproperty!(preconditioner::Chebyshev, f::Symbol, value)` to do that. Let me spin up an PR to do that for `seteigenvalueestimatescaling`

Reply:
Looks like virtually no alteration was needed to support this -- I've gone ahead and removed `setchebyshevtype`. While the example now sets the type in the ctor,

```
chebyshev.type = ChebyshevType.fourth
```

works as well.